doc/source/_toc.yml

```diff
@@ -7,7 +7,37 @@ parts:
       - file: ray-overview/installation
       - file: ray-overview/ray-libraries
 
-  - caption: Ray ML
+  - caption: Ray AI Runtime
+    chapters:
+    - file: ray-air/getting-started
+      title: "What is Ray AI Runtime (AIR)?"
+    - file: ray-air/key-concepts
+    - file: ray-air/user-guides
+      sections:
+        - file: ray-air/check-ingest
+        - file: ray-air/preprocessors
+        - file: ray-air/examples/analyze_tuning_results
+          title: "Analyze hyperparameter tuning results"
+        - file: ray-air/deployment
+        - file: ray-air/examples/upload_to_comet_ml
+        - file: ray-air/examples/upload_to_wandb
+    - file: ray-air/examples/index
+      sections:
+        - file: ray-air/examples/torch_image_example
+        - file: ray-air/examples/tfx_tabular_train_to_serve
```

Comment:
This one's title is rendering as "Set up Ray" in the side-bar. Can you fix the file title, and get rid of the manually specified titles here and elsewhere?

Reply:
Sometimes it can be helpful to have a shorthand title like that, but I agree with the sentiment that getting rid of manual titles is easier to maintain.

Reply:
addressed!